### PR TITLE
Build more wheels

### PR DESCRIPTION
**Pull request by Cadair**
Builds more wheels, is based off #137

### CODE REVIEW

**Review by nabobalis — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -117,9 +123,26 @@ jobs:
       test_command: 'pytest -p no:warnings --doctest-rst --pyargs streamtracer'
       submodules: false
       targets: |
-        - cp3{10,11,12}-manylinux*_x86_64
+        - cp3{10,11,12}-manylinux_aarch64
```
Comment (nabobalis) [inline .github/workflows/ci.yml:126]: I think with this, we cover basically everything (normal)?

File: Cargo.toml [hunk 2]
```diff
@@ -22,3 +22,7 @@ ndarray = {version = "0.15", features = ["rayon"]}
 
 [dev-dependencies]
 float_eq = "1.0.0"
+
+[lints.clippy]
+needless_return = "allow"
+implicit_return = "warn"
```
Comment (nabobalis) [inline Cargo.toml:28]: What made you add this back?

Comment (Cadair) [reply Cargo.toml:28]: I realised that people on newer rust might find it useful in their editor or whatnot.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Cadair) [reply .github/workflows/ci.yml:126]: I think so. Building for ppc etc is rather pointless unless someone asks for it.

**Pull request merged**